Neos.Flow/Tests/Functional/Persistence/Fixtures/SubSubEntityRepository.php

```diff
@@ -11,7 +11,7 @@
  * source code.
  */
 
-use Neos\Flow\Persistence\Repository;
+use Neos\Flow\Persistence\Doctrine\Repository;
```

Comment:
Why? Is the fixture specific for a test related to the Doctrine repository?

Reply:
I think so. As they are only used in the `Neos\Flow\Tests\Functional\Persistence\**Doctrine**\RepositoryTest` they should be of kind doctrine repository.

Reply:
👍